lib/settings/pages/general_settings_page.dart

```diff
@@ -58,6 +58,7 @@ class _GeneralSettingsPageState extends State<GeneralSettingsPage> with SingleTi
   bool showEdgeToEdgeImages = false;
   bool showTextContent = false;
   bool showPostAuthor = false;
+  bool disableScoreCounters = false;
```

Comment:
I feel like the default value for this should be true. We want users to opt-in to seeing the post scores to limit karma farming in Thunder if possible!